vendor/licenses/git_submodule/vscode-tree-sitter-query.dep.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: vscode-tree-sitter-query
+version: d350239cb02d76c4e66720e7b1acb215ec636a18
+type: git_submodule
+homepage: https://github.com/jrieken/vscode-tree-sitter-query.git
+license: none
+licenses: []
+notices: []
```

Comment:
for some reason `./scripts/add-grammar` isn't detecting the MIT license [here](https://github.com/jrieken/vscode-tree-sitter-query?tab=MIT-1-ov-file).

Reply:
This is because the submodule SHA [added](https://github.com/jrieken/vscode-tree-sitter-query/tree/d350239cb02d76c4e66720e7b1acb215ec636a18) doesn't have the licence file which was added a few hours ago. Update the module and it should be found if you manually run this:

https://github.com/github-linguist/linguist/blob/1647b5be1658fb15a5e09bf89067f79b392301e2/script/add-grammar#L135